src/main/cluster-manager.ts

```diff
@@ -59,17 +59,17 @@ export class ClusterManager extends Singleton {
   }
 
   @action protected updateCatalogSource(clusters: Cluster[]) {
```

Comment:
Should we remove whole `updateCatalogSource`  because all localt clusters in the catalog should come from "discovery"? We could migrate existing clusters by adding all kubeconfig files to watch list, right?

Reply:
Yes I think so. Yes we could do that, though since we are already watching `ClusterStore.storedKubeConfigFolder` and `~/.kube/` we probably will get most of them anyway...

Reply:
Will do is separate PR as discussed.